src/SimpleSimulator.cpp

```diff
@@ -281,6 +284,10 @@ int main(int argc, char **argv) {
     // Flags to turn on/off blockwise streaming of input-files
     SimpleSimulator::use_blockstreaming = !(vm["no-streaming"].as<bool>());
 
+    // Flags to turn refetching for streaming of input-files
+    std::cerr << "Prefetching switch off?: " << vm["prefetch-off"].as<bool>() << std::endl;
```

Comment:
You want to keep this print-out?

Reply:
I'd rather like to keep it ^^ But if you would like to remove it, that's fine with mee, too.